tests/pd/mock/mod.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright 2017 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+mod server;
+pub mod case;
```

Comment:
I don't think Case is appropriate here. From the implementation below, I can't consider it as a test case or failure case. It's more readable to call it mocker.